README.md

```diff
@@ -221,22 +221,22 @@ step by setting the sixth argument true, like so:
 $pusher->trigger('my-channel', 'event', 'data', [], true);
 ```
 
-## Authenticating Private channels
+## Authorizing Private channels
```

Comment:
As we already have added user authentication to the codebase. I think it's worth to include it in the README as well.